examples/example.js

```diff
@@ -383,78 +409,90 @@ async function scheduleERC20TransferJob(owner, token, taskCondition) {
       at: {
-        epoches: [Math.floor(new Date().getTime() / 1000 + 3600 ), Math.floor(new Date().getTime() / 1000 + 7200 )]
-      }
-    }
+        epoches: [
+          Math.floor(new Date().getTime() / 1000 + 3600),
+          Math.floor(new Date().getTime() / 1000 + 7200),
+        ],
+      },
+    };
   } else if (process.argv[2] == "schedule-manual") {
     trigger = {
       trigger_type: TriggerType.MANUALTRIGGER,
       manual: true,
-    }
+    };
   }
 
-  const nodeIdOraclePrice  = UlidMonotonic.generate().toCanonical();
-  const nodeIdTransfer     = UlidMonotonic.generate().toCanonical();
+  const nodeIdOraclePrice = UlidMonotonic.generate().toCanonical();
+  const nodeIdTransfer = UlidMonotonic.generate().toCanonical();
   const nodeIdNotification = UlidMonotonic.generate().toCanonical();
 
+  console.log("\nTrigger type", trigger.trigger_type);
+
   const result = await asyncRPC(
     client,
-    'CreateTask',
+    "CreateTask",
     {
       smart_wallet_address: smartWalletAddress,
-      nodes: [{
-        task_type: TaskType.BRANCHTASK,
-        id: nodeIdOraclePrice,
-        name: 'check price',
-        branch: {
-          "if": {
-            expression: `bigCmp(priceChainlink("${config[env].ORACLE_PRICE_CONTRACT}"),toBigInt("10000") > 0`,
-            next: 'transfer_erc20_1'
-          }
-        }
-      }, {
-        task_type: TaskType.CONTRACTWRITETASK,
-        // id need to be unique. it will be assign to the variable
-        id: nodeIdTransfer,
-        // name is for our note only. use for display a humand friendly version
-        name: 'transfer token',
-        contract_write: {
-          // Our ERC20 test token
-          contract_address: config[env].TEST_TRANSFER_TOKEN,
-          call_data: taskBody,
-        }
-      }, {
-        task_type: TaskType.RESTAPITASK,
-        id: nodeIdNotification,
-        name: 'notification',
-        rest_api: {
-          url: "https://webhook.site/fd02e579-a58c-4dbd-8a74-0afa399c0912",
-        }
-      }],
-
-      edges: [{
-        id: UlidMonotonic.generate().toCanonical(),
-        // __TRIGGER__ is a special node. It doesn't appear directly in the task data, but it should be draw on the UI to show what is the entrypoint
-        source: '__TRIGGER__',
-        target: nodeIdOraclePrice,
-      }, {
-        id: UlidMonotonic.generate().toCanonical(),
-        // __trigger__ is a special node. It doesn't appear directly in the task nodes, but it should be draw on the UI to show what is the entrypoint
-        source: nodeIdOraclePrice,
-        target: nodeIdNotification,
-      }],
+      nodes: [
+        {
+          task_type: TaskType.BRANCHTASK,
+          id: nodeIdOraclePrice,
+          name: "check price",
+          branch: {
+            if: {
+              expression: `bigCmp(priceChainlink("${config[env].ORACLE_PRICE_CONTRACT}"),toBigInt("10000") > 0`,
+              next: "transfer_erc20_1",
+            },
+          },
+        },
```

Comment:
This branch node representation is pending comparison and verfication from React Flow. I’ll get the details from frontend.
```
branch: {
  elseIfs: [],
  if: {
    expression: 'bigCmp(priceChainlink("0x694AA1769357215DE4FAC081bf1f309aDC325306"),toBigInt("10000") > 0',
    next: 'transfer_erc20_1'
  },
  else: null
},
```

Reply:
I realized that we could export the entire workflow as json, and here is the branch node’d data. The if else in `conditions`.
```
		{
			"top": -400,
			"left": 864,
			"uuid": "4951011d-0c38-4989-9816-65be605c6a45",
			"options": {
				"conditions": [
					{
						"id": "9dee7ff8-dbe1-4270-9232-94bc3837ffb0",
						"type": "if",
						"statement": "_.isEmpty(fetchDatabase.data)",
						"outgoingPortId": "1faa932e-93dc-4b3e-b553-153258c7f432"
					},
					{
						"id": "32109fbd-4c38-4b71-9a4e-9cc6d824f2d2",
						"type": "if",
						"statement": "",
						"outgoingPortId": "f89ed987-d56f-4ad3-96f1-0779d7cae63a"
					},
					{
						"id": "f09c21e1-c020-4a7a-b8a5-27e7210efacc",
						"type": "if",
						"statement": "",
						"outgoingPortId": "ebb1ed05-8689-4461-89f9-56ff6d3ecb06"
					},
					{
						"id": "640a6318-1173-44e6-bec4-3da179cd5930",
						"type": "else",
						"statement": "",
						"outgoingPortId": "ffc3deaf-76d8-4899-adc7-4918b1310ac6"
					}
				]
			},
			"pluginId": "branch1",
			"blockType": "conditional",
			"editorType": "JavascriptQuery",
			"onboarding": false,
			"environment": "production",
			"isMinimized": false,
			"resourceName": "JavascriptQuery",
			"incomingPorts": [],
			"outgoingPorts": [
				{
					"id": "1faa932e-93dc-4b3e-b553-153258c7f432",
					"name": "0",
					"type": "conditional"
				},
				{
					"id": "f89ed987-d56f-4ad3-96f1-0779d7cae63a",
					"name": "1",
					"type": "conditional"
				},
				{
					"id": "ebb1ed05-8689-4461-89f9-56ff6d3ecb06",
					"name": "2",
					"type": "conditional"
				},
				{
					"id": "ffc3deaf-76d8-4899-adc7-4918b1310ac6",
					"name": "3",
					"type": "conditional"
				}
			],
			"incomingOnSuccessEdges": [
				"c03ec8ae-0cac-4f4b-a2fe-13e21b7cad25"
			]
		}
```

Reply:
```
"conditions": [
					{
						"id": "<ulid>",
						"type": "if",
						"expression": "<expr>",
					},
					{
						"id": "<ulid>",
						"type": "if",
						"expression": "<expr>",
					},
					{
						"id": "<ulid>",
						"type": "else",
						"expression": "<expr>",
					},
]
```
```
edges: [{
    id:
    source: <ulid_of_branch>.<ulid_of_if>
    target: 
}]
```